.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - checkout
       - node/with-cache:
           steps:
-            - run: npm ci
```

Comment:
How come? I thought `ci` was the better option.

Reply:
Because Circle CI restores a cached `node_modules` folder it means `npm ci` was removing the cached `node_modules`. So I think install might be slightly quicker? Not sure though. 

https://app.circleci.com/pipelines/github/btroncone/ngrx-store-localstorage/9/workflows/aea3a4cc-e867-4b42-b8d5-e7440a2bdb49/jobs/9

```
npm ci
npm WARN prepare removing existing node_modules/ before installation
```

Reply:
`npm install` was actually what they had for the default nodejs config, then I change it to `npm ci`, so now I'm putting it back